### PR TITLE
fix(plugin-meetings): add co.webex.com as a valid site

### PR DESCRIPTION
**Pull request by mkesavan13**
# Partially Completes [SPARK-488736 - Fix beta-to-next bugs](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-488736)

## This pull request addresses

- Create meeting is not right for `co.webex` URL. Shows two meeting options upon create with URL in new sample app
- Remote Screenshare on the sample app freezes when we stop share / unpublish in meeting
- Stop Video/Stop Audio cannot send video/audio again after Load Camera/Microphone

### CODE REVIEW

File: packages/@webex/plugin-meetings/src/meeting-info/utilv2.ts [hunk 1]
```diff
@@ -66,6 +68,8 @@ MeetingInfoUtil.isMeetingLink = (value: string) => {
     parsedUrl.pathname &&
     (parsedUrl.pathname.includes(`/${MEET}`) ||
       parsedUrl.pathname.includes(`/${MEET_M}`) ||
+      parsedUrl.pathname.includes(`/${MEET_CISCO}`) ||
```
Comment (sreenara) [inline packages/@webex/plugin-meetings/src/meeting-info/utilv2.ts:71]: Shouldn't we add tests for these changes?

File: docs/samples/browser-plugin-meetings/app.js [hunk 2]
```diff
@@ -1246,34 +1252,88 @@ function handleEffectsButton(btn, type, effect) {
...
+async function stopStartVideo() {
+  if(stopVideoButton.innerText === 'Stop Video') {
+    console.log('MeetingControls#stopVideo()');
+    try {
+      if (localMedia.cameraStream) {
+        localMedia.cameraStream.stop();
+      }
 
-    console.log('MeetingControls#stopVideo() :: Successfully stopped video!');
+      console.log('MeetingControls#stopVideo() :: Successfully stopped video!');
```
Comment (sreenara) [inline docs/samples/browser-plugin-meetings/app.js]: Shouldn't we set the innerText to "Start Video" here? Same comment for the stopStartAudio function

**Review by sreenara — CHANGES_REQUESTED** [review]
Have a couple of comments

File: packages/@webex/plugin-meetings/src/meeting-info/utilv2.ts (hunk 1 above)
Comment (mkesavan13) [reply packages/@webex/plugin-meetings/src/meeting-info/utilv2.ts:71]: As I mentioned in the space, this change is temporary. We might change this again. But if we need test written for the temporary change, I can still go ahead. 

I am following up constantly with Ask App API team in the space for the right way to do it. This change is just until they come back with answers

File: docs/samples/browser-plugin-meetings/app.js (hunk 2 above)
Comment (mkesavan13) [reply docs/samples/browser-plugin-meetings/app.js]: The innerText is changed upon the 'stream-ended' event completion. The change set reflects the same. Please check. 

I should may be change the method name in the log from `stopVideo` to `startStopVideo` and the same for `audio` as well. Will update that.

**Pull request merged**